packages/roosterjs-editor-core/lib/coreAPI/insertNode.ts

```diff
@@ -18,26 +18,30 @@ import {
 } from 'roosterjs-editor-dom';
 
 const insertNode: InsertNode = (core: EditorCore, node: Node, option: InsertOption) => {
-    let position = option ? option.position : ContentPosition.SelectionStart;
-    let updateCursor = option ? option.updateCursor : true;
-    let replaceSelection = option ? option.replaceSelection : true;
-    let insertOnNewLine = option ? option.insertOnNewLine : false;
+    if (!option) {

```

Comment:
This isn't safe because we don't have strict nulls turned on.

Someone could provide an options object that has intentionally nulled values in it. The 4 ternaries guard against that wheras this approach (which reads better) doesn't.